src/js/components/Pagination/stories/Grid.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+
+import { Box, Card, Grid, Grommet, Text } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { Pagination } from '..';
```

Comment:
Can we move this import to line 3, to make sure the component is imported well from the overall package and not just locally? (Relevant for all story examples)

Reply:
Done.